test/organization_test.go

```diff
@@ -32,9 +33,10 @@ func newOrganizationMock(config organizationMockConfiguration) githubcollected.O
 	}
 
 	return githubcollected.Organization{
-		Organization: nil,
-		SamlEnabled:  &samlEnabledMockResult,
-		Hooks:        hooks,
+		Organization:                             nil,
```

Comment:
add test cases for:
1. enterprise not provided and visibility change enabled&disabled
2. enterprise provided and visibility change enabled&disabled